src/styles/blocks/footer.scss

```
@@ -0,0 +1,25 @@
+.footer {
+  position: relative;
+  height: 359px;
+  background-image: url(/images/footer.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: top center;
+
+  &__logo {
+    content: url(/images/logo.png);
```

```suggestion
    background-image: url(/images/logo.png);
```
It's better to use `background-image` for such purposes.
`content` is usually used for `::before` and `::after`